simpleclient_spring_boot/pom.xml

```diff
@@ -64,13 +64,13 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
-            <version>1.3.3.RELEASE</version>
+            <version>1.4.1.RELEASE</version>
```

Comment:
1.4 has only been out a few months, it it reasonable to presume that almost everyone has upgraded already?

I like to keep support for a version that covers the most users, and depend on backwards compatibility for anyone using something newer.